conan/cli/commands/compare.py

```diff
@@ -0,0 +1,134 @@
+import os
+
+from conan.api.output import ConanOutput
+from conan.cli.formatters.compare.compare import format_compare_html, format_compare_txt, \
+    format_compare_json
+from conan.errors import ConanException
+from conan.api.conan_api import ConanAPI
+from conan.cli.command import conan_command
+from conan.api.model import RecipeReference
+
+
+# TODO: TESTING FOR ENCODING ISSUES
+def _execute_command(cmd, stderr=None, ignore_error=False):
```

Comment:
Wondering how we should handle this - We just want to run a git diff and I don't think a copy/paste of an internal function is ok, but we can't use the Git helper either.

Maybe a better idea is to move all/some of this into a new API entrypoint and use the internal version from there?